contribution_inserter/contribution_inserter.ts

```diff
@@ -80,51 +80,43 @@ async function main () {
                 reward_vec.push([u8aToHex(decodeAddress(temporary[k]["account"])), null, temporary[k]["associated_reward"]])
             }
         }
-        calls.push(
-            api.tx.crowdloanRewards.initializeRewardVec(reward_vec)
+        rewardTxs.push(
+            api.tx.scheduler.schedule(args['at-block'] + rewardTxs.length, null, 0, api.tx.crowdloanRewards.initializeRewardVec(reward_vec))
         )
         total_length += temporary.length;
     }
-    calls.push(api.tx.crowdloanRewards.completeInitialization(args["end-relay-block"]));
+    rewardTxs.push(api.tx.scheduler.schedule(args['at-block'] + rewardTxs.length, null, 0, 
+        api.tx.crowdloanRewards.completeInitialization(args["end-relay-block"]))
+    );
+    const batchTx = api.tx.utility.batchAll(rewardTxs);
     
     const account =  await keyring.addFromUri(args['account-priv-key'], null, "ethereum");
     const { nonce: rawNonce, data: balance } = await api.query.system.account(account.address);
     let nonce = BigInt(rawNonce.toString());
 
-    for (let i = 0; i < calls.length; i++) {
+    // We just prepare the proposals
+    let encodedProposal = batchTx?.method.toHex() || "";
```

Comment:
What is `|| ""`?

Reply:
If not valid (according to js, which is very complexe) then use `""`;